charts/stable/bitwardenrs/templates/deployment-ldapsync.yaml

```diff
@@ -30,7 +30,7 @@ spec:
           terminationMessagePolicy: File
           volumeMounts:
             - mountPath: /etc/bitwarden
-              name: {{ if .Values.ldapSync.existingSecret }}{{ .Values.ldapSync.existingSecret }}{{ else }}{{ include "bitwardenrs.name" . }}-ldap{{ end }}
+              name: {{ if .Values.ldapSync.existingSecret }}{{ .Values.ldapSync.existingSecret }}{{ else }}{{ include "bitwardenrs.name" . }}{{ end }}
```

Comment:
I don't use this chart, why is this change and the other one needed?

Reply:
I'm not sure why this question was resolved without an answer? Maybe I missed a chat on the Discord?